draft-ietf-tls-tls13.md

```diff
@@ -3979,7 +3979,7 @@ all exporter computations.
 In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the TLS_AES_128_GCM_SHA256
 cipher suite and SHOULD implement the TLS_AES_256_GCM_SHA384 and
-TLS_CHACHA20_POLY1305_SHA256 cipher suites.
+TLS_CHACHA20_POLY1305_SHA256 cipher suites. (see {{cipher-suites}})
 
```

Comment:
They're bran new suites, so to someone first reading this document, nothing about that has really been stated as of yet. It occurred to me that could be confusing, hence the PR just to add a ref here. It's a big spec; cross-referencing can help, where applicable.